static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTree.tsx

```diff
@@ -291,14 +323,87 @@ export function useVirtualizedTree<T extends TreeLike>(
         }
       }
     },
-    [handleExpandTreeNode, items, tree.flattened]
+    [handleExpandTreeNode, items, tree.flattened, tabIndexKey]
   );
 
+  // Register a resize observer for when the scroll container is resized.
+  // When the container is resized, update the scroll height in our state.
+  // Similarly to handleScroll, we use requestAnimationFrame to avoid overupdating the UI
+  useEffect(() => {
+    if (!props.scrollContainer) {
+      return undefined;
+    }
+    let rafId: number | undefined;
+    const resizeObserver = new window.ResizeObserver(elements => {
+      rafId = window.requestAnimationFrame(() => {
+        dispatch({
+          type: 'set scroll height',
+          payload: elements[0]?.contentRect?.height ?? 0,
+        });
+      });
+    });
+
+    resizeObserver.observe(props.scrollContainer);
+
+    return () => {
+      if (typeof rafId === 'number') {
+        window.cancelAnimationFrame(rafId);
+      }
+      resizeObserver.disconnect();
+    };
+  }, [props.scrollContainer]);
+
+  // Basic required styles for the scroll container
+  const scrollContainerStyles: React.CSSProperties = useMemo(() => {
+    return {
+      height: '100%',
+      overflow: 'auto',
+      position: 'relative',
+      willChange: 'transform',
+    };
+  }, []);
+
+  // Basic styles for the element container. We fake the height so that the
+  // scrollbar is sized according to the number of items in the list.
+  const containerStyles: React.CSSProperties = useMemo(() => {
+    const height = tree.flattened.length * props.rowHeight;
+    return {height, maxHeight: height, overflow: 'hidden'};
+  }, [tree.flattened.length, props.rowHeight]);
+
+  // It is important that this is not executed from a map function because
+  // we are assigning the refs to each individual item. If we do that,
+  // we lose access to the refs and cannot call focus or scrollIntoView on them
```

Comment:
Is this comment still true?

Reply:
Yes, but it's in a bad place and is worded badly